Bio/ExPASy/__init__.py

```diff
@@ -27,11 +26,13 @@ def get_prodoc_entry(id,
     """Get a text handle to a PRODOC entry at ExPASy in HTML format.
 
     >>> from Bio import ExPASy
-    >>> with ExPASy.get_prodoc_entry('PDOC00001') as in_handle:
-    ...     html = in_handle.read()
+    >>> in_handle = ExPASy.get_prodoc_entry('PDOC00001')
```

Comment:
It is a shame this isn't a wrapped handle of some kind, so that we can't use a with statement on Python 2. Oh well, we drop Python 2 support in 2020.